src/Currencies.jl

```diff
@@ -65,7 +65,8 @@ unit(::Currency{S}) where {S} = unit(S)
 code(::Currency{S}) where {S} = code(S)
 name(::Currency{S}) where {S} = name(S)
 
-allsyms()  = keys(_currency_data)
+alldata() = _currency_data
```

Comment:
This is kind of dangerous, as far as encapsulation.
That's why I have the "allpairs", that has a defined interface (an iterator producing pairs of key => values)